File: 240205/특정 조건 배수의 합/sum-of-specific-condition-multiple.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split(" ").map(Number);
let sumVal = 0;

if (input[0] > input[1]) input = [input[1], input[0]];
for (let i = input[0]; i <= input[1]; i++) {
    if (i % 5 == 0) sumVal += i;
}

console.log(sumVal);

File: 231227/특정 조건 배수의 합/sum-of-specific-condition-multiple.py
inp = input().split()
a, b = int(inp[0]), int(inp[1])
sum_val = 0

if a > b:
    a, b = b, a

for i in range(a, b + 1):
    if i % 5 == 0:
        sum_val += i

print(sum_val)